File: app/src/main/java/com/arjanvlek/oxygenupdater/internal/i18n/Locale.java
package com.arjanvlek.oxygenupdater.internal.i18n;

import static com.arjanvlek.oxygenupdater.ApplicationData.LOCALE_DUTCH;

public enum Locale {

    NL, EN;

    public static Locale getLocale() {
        String appLocale = java.util.Locale.getDefault().getDisplayLanguage();

        if(appLocale.equals(LOCALE_DUTCH)) {
            return NL;
        }

        return EN;
    }
}

File: app/src/main/java/com/arjanvlek/oxygenupdater/internal/i18n/Locale.kt
package com.arjanvlek.oxygenupdater.internal.i18n

import com.arjanvlek.oxygenupdater.ApplicationData

enum class Locale {
    NL,
    EN;

    companion object {
        val locale: Locale
            get() {
                return if (java.util.Locale.getDefault().displayLanguage == ApplicationData.LOCALE_DUTCH) NL else EN
            }
    }
}
